lib/l10n/intl_en.arb

```diff
@@ -66,6 +62,22 @@
       }
     }
   },
+  "cardsDemoNotSelected": "Not selected",
+  "@cardsDemoNotSelected": {
+    "description": "Indicates the status of a (selectable) card being not selected"
+  },
+  "cardsDemoSelect": "Select",
+  "@cardsDemoSelect": {
+    "description": "Select a (selectable) card"
+  },
+  "cardsDemoSelectable": "Selectable (long press)",
+  "@cardsDemoSelectable": {
+    "description": "If the user taps and hold this card. The card will toggled between on and off."
+  },
+  "cardsDemoSelected": "Selected",
+  "@cardsDemoSelected": {
+    "description": "Indicates selected status of a (selectable) card"
+  },
```

Comment:
suggest making these more generic e.g. "notSelected" instead of "cardsDemoNotSelected", since we may want to use these for other selectable things in the future.